pymongo/encryption_options.py

```diff
@@ -219,3 +219,44 @@ def __init__(
         # Maps KMS provider name to a SSLContext.
         self._kms_ssl_contexts = _parse_kms_tls_options(kms_tls_options)
         self._bypass_query_analysis = bypass_query_analysis
+
+
+class EncryptionRangeOpts:
+    """Options to configure encrypted queries using the rangePreview algorithm."""
+
+    def __init__(
+        self,
+        sparsity: int64,
```

Comment:
I think it's fine if we say int here and still do the int -> int64 conversion internally.